UnitTests/AggregatePixelTests.swift

```diff
@@ -47,13 +47,14 @@ class AggregatePixelTests: XCTestCase {
             await subject1.incrementAndSendIfNeeded() // Concurrency will increment the counter but prevent this firing
         }
 
-        XCTAssertEqual(1, pixel.pixels.count)
-        XCTAssertEqual("1", pixel.pixels[0].params?["count"])
```

Comment:
This has been crashing for me as `pixel.pixels` was empty at the time of checking. The line above was causing a failure. If this shouldn't be the case, the test needs to be updated. I'm not familiar with the code though so I'd appreciate some help.